reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerMetricsRecorder.java

```diff
@@ -54,4 +54,22 @@
 	 * @param time the total time in nanoseconds for the request/response
 	 */
 	void recordResponseTime(String uri, String method, String status, Duration time);
+
+	/**
+	 * Record a new active in-use http connection
+	 *
+	 * @param uri the requested URI
+	 * @param method the HTTP method
+	 * @since 1.0.15
+	 */
+	default void recordServerConnectionActive(String uri, String method) { }
```

Comment:
as discussed off-band, let's avoid tagging with uri/method, and align input parameters with `recordServiceConnectionOpened|Closed`: only provide `SocketAddress localAddress`.